# This is a message for storing a 2D object detection and corresponding 3D point description
yeet/BoundingBox bounding_box
yeet/ObjectDescription description